### PR TITLE
github action for black linting checked added

### LINKED ISSUE #11: Add linting checks for github actions.
We would like to use github actions to test incoming PRs for styling/formatting/type checking issues before made to review.

### PR DESCRIPTION
**Pull request by shaharyar-shamshi**
fixes #11

### CODE REVIEW

**Review by lullis — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+black==22.10.0
```
[the remark this answers is not in the document]
Comment (shaharyar-shamshi) [reply requirements.txt]: @lullis  fixed. I have added proper action and tested it as well

**Comment by shaharyar-shamshi** [comment]
@lullis  should I add branch as well to trigger the action

**Review by lullis — APPROVED** [review]

**Comment by lullis** [comment]
@shaharyar-shamshi , you need to sign your commit in order for it to be merged.

**Comment by shaharyar-shamshi** [comment]
@lullis done

**Review by lullis — APPROVED** [review]

**Pull request merged**